db/migrate/20160824115000_add_parent_file_to_video_files.rb

```diff
@@ -0,0 +1,6 @@
+class AddParentFileToVideoFiles < ActiveRecord::Migration
+  def change
+    add_column :pageflow_video_files, :parent_file_id, :integer
+    add_column :pageflow_video_files, :parent_file_model_type, :string
+  end
```

Comment:
This means we need to add the index to every file type table including panoramas, right?